src/assets/i18n/en.json

```
@@ -424,6 +424,17 @@
         "title": "Telemetry",
         "label": "Send anonymous usage data",
         "description": "The data that will be sent is limited to the version of Oyasumi that you are using, and the language you have set. No other data is sent and it is completely anonymous. This data is only used for providing some general usage statistics."
+      },
+      "systemTray": {
+        "title": "System tray",
+        "close": {
+          "label": "Exit in system tray",
```

Imo a bit more clear, as it doesn't actually exit
```suggestion
          "label": "Close to system tray",
```